arch/arm/core/offsets/offsets_aarch64.c

```diff
@@ -22,8 +22,13 @@
  * completeness.
  */
 
+#ifndef _ARM_OFFSETS_INC_
+#ifndef _ARM_OFFSETS_INC_
```

Comment:
That doesn't look quite right.  Maybe also rename to ARM64.

Reply:
Copy and paste error... Fixed.

I intentionally named it the same as in AARCH32 so that only one offset file can be included at max... though probably I am being paranoid.